### ISSUE TITLE
Failed to download file

### ISSUE THREAD

**Issue by clarkrs481** [rmcrackan/Libation#191]
All books downloaded except one, which gives an error.
[Log202201.log](https://github.com/rmcrackan/Libation/files/7937492/Log202201.log)

**Comment by rmcrackan** [comment]
Are you able to play [this book](https://www.audible.com/pd/B07DKTRC2Z) on your phone?

Libation imported it but this error looks like it's no longer part of your audible library. Was it returned or somehow removed?

**Comment by clarkrs481** [comment]
Yup.  Plays on phone and in Windows 10 player and still appears and plays in Library on Audible.com.  I've never returned any book.

**Comment by rmcrackan** [comment]
This is bizarre. Let's see if Verbose logging can shed some light.

* Open Libation settings
* Change Logging level from `Information` to `Verbose`
* Restart Libation
* Try again
* [Email me](https://github.com/rmcrackan) your new log
* Change Logging level back to `Information`

About Verbose logging: only recommended for debugging tough problems. After the log is created which contains our bug, reset Logging level to Information. I've tried not to let any personal info get into logs. This is harder to guarantee with verbose. This is also why I recommend emailing instead of posting here publicly -- just in case.

**Comment by rmcrackan** [comment]
From email conversation:

> I looked further into this on Audible.com and I was able to directly download an mp4 from there, so I'm wondering if they just don't have a lossless version for some reason.  Downloading separately isn't something I was doing previously, so that may be normal and I'm just reading something into it that isn't there.

Thanks for the emailed log. I've never seen this before. Shy of buying this book myself for testing. I'm not sure how to debug this one weird case.

Since you're able to get this by other means, I'm not going to rush to fix this bug and I'm going to keep this issue open so I don't forget about it.
